build.sh

```diff
@@ -42,9 +40,15 @@ usage()
     echo "skipnative - do not build native components."
     echo "skipmscorlib - do not build mscorlib.dll."
     echo "skiptests - skip the tests in the 'tests' subdirectory."
-    echo "skiprestore - skip restoring nuget packages."
     echo "skipnuget - skip building nuget packages."
-    echo "disableoss - Disable Open Source Signing for mscorlib."
+    echo "verbose - optional argument to enable verbose build output."
+    echo "-skiprestore: skip restoring packages ^(default: packages are restored during build^)."
+	echo "-disableoss: Disable Open Source Signing for System.Private.CoreLib."
+	echo "-priority=^<N^> : specify a set of test that will be built and run, with priority N."
```

Comment:
I don't believe you need the 'priority' arg here, since we don't build tests on non-windows yet